.gitignore

```diff
@@ -1 +1,2 @@
 *.swp
+*~
```

Comment:
Why add this to the .gitignore?

Reply:
This is for emacs temp files.  They get created every time it modifies a file.

Reply:
I removed it and added it to my global gitignore (which I should have done in the first place).